src/sql/src/session/vars.rs

```diff
@@ -539,6 +539,20 @@ const ENABLE_UPSERT_SOURCE_DISK: ServerVar<bool> = ServerVar {
     safe: true,
 };
 
+static DEFAULT_UPSERT_ROCKSDB_TUNING_CONFIG: Lazy<String> = Lazy::new(|| "{}".to_string());
+/// Tuning for RocksDB used by `UPSERT` sources that takes effect on restart.
```

Comment:
Is this something that you want to add for running experiments or something that you intend to expose to support & oncall engineers? Do we have an idea the RocksDB parameters that are most likely to be tuned?

Reply:
Possible for oncall engineers!

Yes, in the `src/rocksdb/src/tuning.rs` we have the important ones